lab-bundle-spec.md

```diff
@@ -135,21 +143,18 @@ The properties of each environment resource will depend on their type, i.e. AWS
 attribute | required | type   | notes
 --------- | -------- | ------ | -----------------------------------------
 type      | ✓        | enum   | [See list of valid types]
-id        |          | string | Identifier that can be used throughout project bundle
+id        |          | string | Identifier that can be used throughout project bundle.
+variant   |          | string | The subtype resource being requested. Each type below lists its valid variants and specifies which is the default.
 
 ```yml
 environment_resources:
   - type: gcp_project
     id: my_primary_project
-    dm_template:
-      script: deployment_manager
+  - type: gcp_project
+    id: secondary_project
+    variant: gcpfree
   - type: gcp_user
     id: primary_user
-    permissions:
-      - project: my_primary_project
-        roles:
-          - roles/editor
-          - roles/bigquery.admin
   - type: gcp_user
```

Comment:
Nit: I know we haven't really talked about it yet, but "GCP user" is a misnomer and this resource is really a Gaia User. Given that v2 includes breaking changes from V1, I'd like to rename this to `gaia_user`.

To make upgrading easier, we can also start supporting `gaia_user` in the V1 spec and then doa  large find+replace on all V1 content so creators don't have to think about this when upgrading to V2.

Reply:
I'm fine with changing this to `gaia_user` - @jgan and @dcbrandao sound ok to you?

Reply:
technically sam is correct, but 'gaia user' is a super confusing term. if i google for that, i get directed to a registration page for an online game

Reply:
agree with @dcbrandao

Reply:
@supersam654 are you ok with leaving it as `gcp_user` given Derek and Dave's comments?

Reply:
I don't feel super strongly either way so `gcp_user` is fine. With that said, I'd at least like a comment explaining why we're sticking with the misnomer.

Reply:
Added "The `gcp_user` type could more properly be called `gaia_user`, since that's what it provisions. However, the term `gaia` is less well-known, so we stick with `gcp`." below